integrations/mongodb-integration.mdx

```diff
@@ -3,7 +3,7 @@ title: "MongoDB"
 description: MongoDB Integration - Setup & Installation Docs | Middleware
 ---
 
-Collects MongoDB performance and health metrics
+The MySQL Integration allows users to ingest and monitor the performance and health metrics of their NoSQL database instances. This integration is useful for tracking the total number of active connections and operations executed within your instances, the total space allocated to database indexes, the count of all document operations executed, and much more. 
```

Comment:
Curious, where did NoSQL come from? MySQL is inherently SQL, and NoSQL is a term for graphical and non relational databases.

Reply:
OH I see where. This is the mongodb doc but you say MySQL integration in here. That's the real typo.